.codex/hooks/format_after_write.py

```
@@ -0,0 +1,189 @@
+    ]
+    paths = []
+    for candidate in candidates:
+        path = normalize_path(root, candidate)
+        if path is not None:
+            paths.append(path)
+    return paths
+
+
+def changed_paths(root: pathlib.Path) -> list[pathlib.Path]:
+    result = subprocess.run(
+        ["git", "diff", "--name-only"],
```

Ruff (0.15.12)

[error] Starting a process with a partial executable path

(S607)